apps/publisher/app-scripts/init-scripts/app.js

```diff
@@ -39,6 +39,8 @@ var pubConfig = require('/config/publisher.js').config();
 var mod = require('store');
 var rxt=require('rxt');
 var lifecycle=require('lifecycle');
+var metricsAPI = require('carbon-metrics').metrics;
+
```

Comment:
Thanks @lahirus , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/cd6cd2f172c69eedd1e370ccc6880f0079ff803d)